assignments/prototypes.js

```diff
@@ -32,6 +59,23 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+function Humanoid(atttributes){
+  GameObject.call(this,atttributes);
```

Comment:
You don't have to use `.call` with `GameObject` because the `CharacterStates` constructor function (which `Humanoid` is calling into its context) already does it inside its own function definition.